manifest_staging/charts/csi-secrets-store-provider-azure/arc-values.yaml

```diff
@@ -171,6 +171,54 @@ constructPEMChain: true
 
 # Azure Arc Extension
 enableArcExtension: true
+promMdmConverter:
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+    limits:
+      cpu: 50m
+      memory: 100Mi
+mdm:
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+    limits:
+      cpu: 50m
+      memory: 100Mi
+msiAdapter:
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+    limits:
+      cpu: 50m
+      memory: 100Mi
+telegraf:
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+    limits:
+      cpu: 50m
+      memory: 100Mi
+amacoreagent:
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+    limits:
+      cpu: 50m
+      memory: 100Mi
+fluentd:
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+    limits:
+      cpu: 50m
+      memory: 100Mi
```

Comment:
For `fluentd`, the current hardcoded limits are

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/ecaccb15f119e8c2cd47be70cf602e1f38eb307a/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml#L184-L193

Is `100Mi` sufficient?

Reply:
This is the default limit we decided to set based on our provider pod defaults. It's works for most users. Making it configurable would help with edge cases scenarios where user need to increase them due to higher consumption.

Reply:
In this PR we're changing to default to `100Mi` from the previous `250Mi`. Is that intended?

Reply:
reverted to original defaults.